helpers/logging

```diff
@@ -186,6 +186,35 @@ ynh_exec_fully_quiet() {
     fi
 }
 
+# Execute a command and redirect stderr in /dev/null. Print stderr on error.
+#
+# usage: ynh_exec_stderr_on_error your command and args
+# | arg: command - command to execute
+#
+# Note that you should NOT quote the command but only prefix it with ynh_exec_stderr_on_error
+#
+# Requires YunoHost version 3.2.0 or higher.
+ynh_exec_stderr_on_error() {
+    logfile="$(mktemp)"
+    rc=0
+    # Boring legacy handling for when people calls ynh_exec_* wrapping the command in quotes,
+    # (because in the past eval was used) ...
+    # we detect this by checking that there's no 2nd arg, and $1 contains a space
+    if [[ "$#" -eq 1 ]] && [[ "$1" == *" "* ]]
```

Comment:
Imho there's no legacy to be handled here because it's a new command, and we should just assume people will use it in the appropriate way, ie just calling `ynh_exec_stderr_on_error foo --bar`